### DISCUSSION TITLE
How to create a filtered view

### DISCUSSION THREAD

**Discussion by rossmacarthur** [aminalaee/sqladmin#615] [Q&A]
For example let's say I have a `Wallet` and a list of `Transaction` associated with that wallet. How would I be able to see the list of transaction just for one wallet? Ideally I would be able to click from a `Wallet` page to see the transactions.

**Comment by aminalaee** [comment]
Can you provide a minimal example of what you want to do?
Doesn't it by default link to Transaction page from Wallet page?

**Comment by johanboekhoven** [comment]
@rossmacarthur I think the easiest way could be to create a columnformatter that builds a custom URL with the correct filter arguments for for example the id column. So clicking on that link would get you to the listview of Transaction with a set of filter rules based on the Wallet ID, provided you've stored wallet_id as field on the Transaction.

**Reply by aminalaee** [reply]
That's a very reasonable use-case 👍
